### DISCUSSION TITLE
hospitalization data interpretation

### DISCUSSION THREAD

**Discussion by naczek** [owid/covid-19-data#2815] [Q&A] [answered]
Hi,
As the documentation provides, the hospitalization data column contains the summed values for each week. What if I need to calculate the cumulative number of hospitalized people, namely the running total? Should I sum up every 7th value? Or is there any raw data, I could use?
Thank you

**Comment by lucasrodes** [comment, marked as answer]
Hi @naczek,
Thanks for your interest in our data.

While some countries report this value once a week (you'd be safe to sum them all up), others provide a 7-day average (e.g. Germany).

I'd suggest applying a 7-day window, averaging the values to get a single value and then perform the cumulative sum. Alternatively, you can also get the 7th value, but I am unsure about the regularity in the reporting (maybe sometimes a day is skipped etc.)

Note that this method is an approximation. Also, same people can be hospitalised multiple times, so you could be double-counting people too.

**Reply by naczek** [reply]
Thank you!
